src/main/java/org/tron/core/db/Manager.java

```diff
@@ -97,6 +98,9 @@
 @Component
 public class Manager {
 
+  public static ThreadLocal<Boolean> isProcessBLock = new ThreadLocal<>();
+  public static ThreadLocal<BlockCapsule> currentBlcok = new ThreadLocal<>();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make currentBlcok a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1471&issues=AWW0lDYB_eEZPtUmTRSc&open=AWW0lDYB_eEZPtUmTRSc)

[ISSUE_KEY]: AWW0lDYB_eEZPtUmTRSc